lib/config.js

```diff
@@ -181,8 +182,9 @@ function parseED25519 (prefix) {
       throw new ServerError('No ' + (prefix ? prefix.toUpperCase() + '_' : '') +
         'ED25519_SECRET_KEY provided.')
     }
-    keyPair = tweetnacl.sign.keyPair()
-    ed25519.secret = tweetnacl.util.encodeBase64(keyPair.secretKey)
+    const seed = crypto.randomBytes(32)
+    keyPair = tweetnacl.sign.keyPair.fromSeed(seed)
+    ed25519.secret = seed.toString('base64')
```

Comment:
In profiling we noticed that the tweetnacl base64 encoding seemed slow, so I'm glad you used the node built-in. Should we do the same for the public key below?